x-pack/plugins/stack_connectors/public/connector_types/bedrock/params.tsx

```diff
@@ -102,7 +102,7 @@ const BedrockParamsFields: React.FunctionComponent<ActionParamsProps<BedrockActi
       >
         <EuiFieldText
           data-test-subj="bedrock-model"
-          placeholder={'anthropic.claude-v2'}
```

Comment:
Noted in the function tests (below) that it would perhaps be good to add a test for the old default, which some customers may have used explicitly, and could be used for every future version of Kibana till the ends of time ...